teamcity/resource_build_trigger_schedule.go

```diff
@@ -173,6 +173,11 @@ func resourceBuildTriggerSchedule() *schema.Resource {
 				Optional: true,
 				Default:  "<default>",
 			},
+			"env_params": {
```

Comment:
I am wondering if to be consistent with TeamCity's terminology and UI, should we add "build_customization" and "env_params", "enforce_clean_checkout",... will be sub-fields. What do you think?

(This would make it backward incompatible though.)

Reply:
We can but I don't see much benefit of it since it's making the input a bit more user unfriendly. Personally I think enforce_clean_checkout and enforceCleanCheckoutForDependencies fits quite well along with other properties, not sure why TeamCity changed the API. Also the name `env_params` is already adding an extra layer of translation. What do you think?